Podfile

```diff
@@ -2,11 +2,11 @@ platform :ios, '8.0'
 use_frameworks!
 
 target 'Connect' do
-  pod 'MonkeyKitUI', :path => '~/Workspace/Monkey-UI-iOS'
+  pod 'MonkeyKitUI', :path => '~/Documents/git Shippify/Monkey-UI-iOS-master'
```

Comment:
Please change it so it points to ` :git => https://github.com/Criptext/Monkey-UI-iOS`